hack/doc/role-permissions2md.py

```diff
@@ -0,0 +1,150 @@
+import sys
+from urllib.parse import urlparse
+import yaml
+import argparse
+from urllib.request import urlopen
+
+apiTerms = {
```

Comment:
I think it's enough to capitalize the first letter of the word using, `title()` or `capitalize()`, functions. We don't need all these lookup tables to get the right case of each of these words.

Reply:
The generated tables are going into the official documentation. They should look as nice as possible. For the `apiTerms`, I agree, but resource names should be "handcrafted".

Reply:
I disagree, capitalize the first letter and be done with it

Reply:
We would have stuff like `Mutatingwebhookconfigurations`. That's just ugly an doesn't look professional in our official documentation. Won't be doing that.

Reply:
Then we can keep it all lowercase, since that's how the resource names are in the actual manifests 🙂